types/mithril/index.d.ts

```diff
@@ -294,5 +294,197 @@ declare namespace Mithril {
 	}
 }
 
+declare global {
+    namespace JSX {
+        // tslint:disable-next-line:no-empty-interface
+        interface Element extends Mithril.Vnode { }
+
+        // tslint:disable-next-line:no-empty-interface
+        interface IntrinsicAttributes extends Mithril.Attributes { }
+        // tslint:disable-next-line:no-empty-interface
+        interface IntrinsicClassAttributes extends Mithril.Attributes { }
+
+        interface IntrinsicElements {
```

Comment:
Forgot to mention: please include the various special vnode types (especially `"["` for fragments) in this list. Those are pretty important.

Reply:
how can I do this? adding "[" as a key and pointing to `Mithril.Attributes`? what are the other special vnode types?

Reply:
Yes. You literally add those to the list.